### PR TITLE
fix: grey out a completed task

### LINKED ISSUE #9: "gray out" a completed task
Currently, when the user marks a task as "complete", the only changes on the UI is that the content text of the task gets a line through. A more impactful change needs to be done, changing the color of the panel to a more darker one seems appropiate.

### PR DESCRIPTION
**Pull request by simonhoyos**
# Gray out a completed task

**Current view**
Screen Shot 2020-10-27 at 16 52 33

**After changes**
Screen Shot 2020-10-27 at 16 56 05

closes #9

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ document.addEventListener("DOMContentLoaded", () => {
 
         // Message container
         const message = document.createElement("article");
-        message.classList.add("message", "is-primary");
+        message.classList.add("message", done ? "is-secondary" : "is-primary");
```
Comment (DevTony101) [inline src/index.js:59]: There is no "is-secondary" color class in bulma. Perhaps you meant to say "is-dark"?

File: src/index.js [hunk 2]
```diff
@@ -147,6 +147,8 @@ document.addEventListener("DOMContentLoaded", () => {
     const id = Number(task.dataset.id);
     database.toggleDone(id, isDone, () => {
       task.classList.toggle("is-done", isDone);
+      task.classList.toggle("is-primary", !isDone);
+      task.classList.toggle("is-secondary", !isDone);
```
Comment (DevTony101) [inline src/index.js:151]: There is no "is-secondary" color class in bulma. Perhaps you meant to say "is-dark"? 

Also, there seems to be a little bug here, putting "!isDone" in both toggle functions will remove and add both classes at the same time.

**Review by DevTony101 — CHANGES_REQUESTED** [review]
Thank you so much for your contribution @simonhoyos ! But there are few things you missed that I commented out. Please feel free to correct them and re-submit your pull request :)

File: src/index.js (hunk 1 above)
Comment (simonhoyos) [reply src/index.js:59]: Thanks for pointing this out. I'll fix it

**Comment by DevTony101** [comment]
Hi there! Although I appreciate your contribution this issue was just resolved by #10, so I'm afraid I must close this PR. But please don't let this discourage you to open a new issue for a new exciting feature you want to add 👍🏽

**Comment by simonhoyos** [comment]
To bad as this issue was assigned to me 👎

**Comment by DevTony101** [comment]
I'm sorry! It is not that I didn't take that into account, it just seemed unfair to the other person who committed twice to solve the issue.

**Pull request closed without merging**